### PR TITLE
feat: create opacity props on PSkeleton

### PR DESCRIPTION
**Pull request by minseolee**
## To Reviewers
- [ ] Skipping reviews is not a problem

## Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### CODE REVIEW

File: src/feedbacks/loading/skeleton/PSkeleton.stories.mdx [hunk 1]
```diff
@@ -111,6 +111,24 @@ import PSkeleton from "@/feedbacks/loading/skeleton/PSkeleton";
             type: 'text',
         }
     },
+    opacity: {
+        name: 'opacity',
+        type: {name: 'number'},
+        description: 'Opacity of skeleton',
+        defaultValue: '0.4',
```
Comment (song2kim) [inline src/feedbacks/loading/skeleton/PSkeleton.stories.mdx:118]: The type is expected number, but it is entered as a string, so please check it

**Review by WANZARGEN — APPROVED** [review]
LGTM! :)

File: src/feedbacks/loading/skeleton/PSkeleton.stories.mdx [hunk 2]
```diff
@@ -111,6 +111,24 @@ import PSkeleton from "@/feedbacks/loading/skeleton/PSkeleton";
...
+        table: {
+            type: {
+                summary: 'number'
+            },
+            category: 'props',
+            defaultValue: {
+                summary: 'number'
```
Comment (WANZARGEN) [inline src/feedbacks/loading/skeleton/PSkeleton.stories.mdx:125]:
```suggestion
                summary: '0.4'
```

**Review by WANZARGEN — APPROVED** [review]
I missed a comment for story args. please check!

File: src/feedbacks/loading/skeleton/PSkeleton.stories.mdx (hunk 2 above)
Comment (minseolee) [reply src/feedbacks/loading/skeleton/PSkeleton.stories.mdx]: Thanks a lot. I've approved it

**Review by song2kim — APPROVED** [review]
LGTM :-)

**Pull request merged**